apps/errant-tower-client/src/common/styles/mixins.scss

```
@@ -0,0 +1,61 @@
+        cursor: pointer;
+        color: $color-dark-400;
+        background: $color-primary-600;
+    }
+
+    &:disabled {
+        opacity: 0.8;
+        filter: grayscale(0.8);
+    }
+}
+
+@mixin primaryDivider {
```

Stylelint (17.9.0)

[error] Expected mixin name "primaryDivider" to be kebab-case (scss/at-mixin-pattern)

(scss/at-mixin-pattern)